docs/natspec-format.rst

```diff
@@ -0,0 +1,202 @@
+Solidity contracts can have a special form of comments that form the
+basis of the Ethereum Natural Specification Format.
+
+.. _header-n3:
+
+Documentation Example
+=====================
+
+Documentation is inserted above each ``class``, ``interface`` and
+``function`` using the doxygen notation format.
+
+-  For Solidity you may choose ``///`` for single or multi-line
+   comments, or ``/**`` and ending with ``*/``.
+
+-  For Vyper, use ``"""`` indented to the inner contents with bare
+   comments. See `Vyper
+   documentation <https://vyper.readthedocs.io/en/latest/structure-of-a-contract.html#natspec-metadata>`__.
+
+This example shows a contract and a function using all available tags.
+Note: NatSpec currently does NOT apply to variables (see
+`solidity#3418 <https://github.com/ethereum/solidity/issues/3418>`__),
+even if they are declared public and therefore do affect ABI. Note:
+NatSpec currently only interprets tags functions if they are external or
+public. You are welcome to use similar comments for your internal and
+private functions, but those will not be parsed.
+
+.. code:: solidity
+
+   pragma solidity ^0.5.1;
+
+   /// @title A simulator for Bug Bunny, the most famous Rabbit
+   /// @author Warned Bros
+   /// @notice You can use this contract for only the most basic simulation
+   /// @dev All function calls are currently implement without side effects
+   contract BugsBunny {
```

Comment:
I'm not sure we should use trademarks (I assume) in the documentation without the need for them.